### PR TITLE
Hotfix/FP-1332: Horz. Scrollbar Caused by Sections

### PR DESCRIPTION
**Pull request by tacc-wbomar**
## Suggested

- to also fix UTRC, https://github.com/TACC/Core-CMS-Resources/pull/108

## Overview

Avoid horizontal scrollbar on pages known to use sections. (I am the only one adding sections, until [FP-1318](https://jira.tacc.utexas.edu/browse/FP-1318).)

## Issues

[FP-1332](https://jira.tacc.utexas.edu/browse/FP-1332)

## Testing

1. Load these pages to test:
    - https://pprd.frontera-portal.tacc.utexas.edu/system [^0] (uses Core Standard template)
    - https://pprd.utrc.tacc.utexas.edu/ [^0] (uses UTRC Home template)
1. Verify window has **no** horizontal scrollbar.

[^0]: Deployed via https://jenkins01.tacc.utexas.edu/job/Core_Portal_Deploy/849/ which loads image from branch `main--v3-3-0-with-hotfixes`.

## Notes

This is not a long-term solution. Via [FP-1318](https://jira.tacc.utexas.edu/browse/FP-1318), I will fix the problem at the source (I do not know how yet, but I'll figure it out).

### CODE REVIEW

File: taccsite_cms/templates/standard.html [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+  {{ block.super }}
+
+  {# TODO: FP-1318: Solve section overflow in `o-section` code #}
+  <style>
+  /* To hide `.o-section--style-…` and `.o-section__banner-image` overflow */
+  /* FAQ: Isolate from other templates to avoid unexpected side-effects */
+  main,
+  /* HACK: Using ID until upstream updates markup via `quick/create-and-document-element-stylesheets` */
+  #content-wrap { overflow: hidden; }
+  </style>
```
Comment (tacc-wbomar) [inline taccsite_cms/templates/standard.html:14]: This fix is already working for Frontera home and can fix UTRC home. See https://github.com/TACC/Core-CMS-Resources/pull/108/files.

**Review by edmondsgarrett — APPROVED** [review]
LGTM

**Review by taoteg — APPROVED** [review]
LGTM.

**Comment by tacc-wbomar** [comment]
Thanks, y'all.

**Pull request merged**